examples/run_Hahn_echo.py

```diff
@@ -31,127 +40,95 @@
-        echo_data.C.mean("nScans"))
-    echo_data.reorder("t2", first=False)
-    for_plot = echo_data.C
-    for_plot.ft('t2',shift=True)
-    fl.next('FTed data')
-    fl.image(for_plot)
-echo_data.name(config_dict["type"] + "_" + str(config_dict["echo_counter"]))
-echo_data.set_prop("postproc_type", "proc_Hahn_echoph")
+echo_data.chunk(
+    "t",
+    ["ph1", "t2"],
+    [len(ph1_cyc), -1],
+)
+echo_data.setaxis("ph1", ph1_cyc / 4)
+echo_data.reorder(["ph1", "nScans", "t2"])
+echo_data.set_prop("postproc_type", "spincore_SE_v1")
+echo_data.set_prop("coherence_pathway", {"ph1": +1})
 echo_data.set_prop("acq_params", config_dict.asdict())
-target_directory = getDATADIR(exp_type="ODNP_NMR_comp/Echoes")
+nodename = config_dict["type"] + "_" + str(config_dict["echo_counter"])
+echo_data.name(nodename)
 filename_out = filename + ".h5"
-nodename = echo_data.name()
 if os.path.exists(f"{filename_out}"):
     print("this file already exists so we will add a node to it!")
     with h5py.File(
         os.path.normpath(os.path.join(target_directory, f"{filename_out}"))
     ) as fp:
-        if nodename in fp.keys():
-            print("this nodename already exists, so I will call it temp_echo")
-            echo_data.name("temp_echo")
-            nodename = "temp_echo"
-    echo_data.hdf5_write(f"{filename_out}", directory=target_directory)
-else:
-    try:
-        echo_data.hdf5_write(f"{filename_out}", directory=target_directory)
-    except:
-        print(
-            f"I had problems writing to the correct file {filename}.h5, so I'm going to try to save your file to temp_echo.h5 in the current directory"
-        )
-        if os.path.exists("temp_echo.h5"):
-            print("there is a temp_echo.h5 already! -- I'm removing it")
-            os.remove("temp_echo.h5")
-            echo_data.hdf5_write("temp_echo.h5")
-            print(
-                "if I got this far, that probably worked -- be sure to move/rename temp_echo.h5 to the correct name!!"
-            )
+        while nodename in fp.keys():
+            nodename = config_dict["type"] + "_" + str(config_dict["echo_counter"])
+            data.name(nodename)
+            config_dict["echo_counter"] += 1
+echo_data.hdf5_write(f"{filename_out}", directory=target_directory)
 print("\n*** FILE SAVED IN TARGET DIRECTORY ***\n")
-print(("Name of saved data", echo_data.name()))
-config_dict.write()
 print(
     "Your *current* γ_eff (MHz/G) should be ",
     config_dict["gamma_eff_MHz_G"],
     " - (Δν*1e-6/",
-    Field,
+    field_G,
     "), where Δν is your resonance offset",
 )
 print(
-    "Now run Hahn_echo_proc.py to see your spectra and offset, that script will correct your gamma effective for you!"
+    "saved data to (node, file, exp_type):",
+    echo_data.name(),
+    filename_out,
+    my_exp_type,
+)
+config_dict.write()
+# }}}
+subprocess.call(
+    ["python", "examples/proc_raw.py", echo_data.name(), filename_out, my_exp_type],
+    cwd=r"C:\git\proc_scripts",
```

Comment:
you're not understanding the code that I gave you

Reply:
I'm using subprocess in a different way but this should still work